src/pages/index.js

```diff
@@ -36,24 +59,103 @@ const profileEditFormPopup = new PopupWithForm({
 
 profileEditFormPopup.setEventListeners();
 
+api
+  .getUserInfo()
+  .then((userData) => {
+    profileUserInfo.setUserInfo({
+      name: userData.name,
+      description: userData.about,
+    })
+    profileUserInfo.setAvatarInfo({
+      link: userData.avatar,
+    })
+  })
```

Comment:
You need to `catch` possible errors at the very **end** of any server request.

**Please, correct it everywhere in the code**